hazelcast/src/test/java/com/hazelcast/replicatedmap/ReplicatedMapMergePolicyTest.java

```diff
@@ -77,30 +69,38 @@ public void killAllHazelcastInstances() {
     @Parameterized.Parameter
     public ReplicatedMapMergePolicyTestCase testCase;
 
+    private TestHazelcastInstanceFactory factory;
```

Comment:
Ah, forget it, sorry. I misread the `createHazelcastInstanceFactory(2)` as `new TestHazelcastInstanceFactory()`.